cockatrice/src/window_main.cpp

```diff
@@ -79,7 +83,7 @@ void MainWindow::processConnectionClosedEvent(const Event_ConnectionClosed &even
             break;
         }
         case Event_ConnectionClosed::SERVER_SHUTDOWN: reasonStr = tr("Scheduled server shutdown."); break;
-        case Event_ConnectionClosed::USERNAMEINVALID: reasonStr = tr("Invalid username.\nYou may only use A-Z, a-z, 0-9, _, ., and - in your username."); break;
+        case Event_ConnectionClosed::USERNAMEINVALID: reasonStr = tr("Invalid username."); break;
```

Comment:
I think it needs to be there for the compiler

On Wed, Jul 1, 2015 at 6:45 PM Zach <notifications@github.com> wrote:

> In cockatrice/src/window_main.cpp
> <https://github.com/Cockatrice/Cockatrice/pull/1180#discussion_r33732908>:
>
> > @@ -79,7 +83,7 @@ void MainWindow::processConnectionClosedEvent(const Event_ConnectionClosed &even
> >              break;
> >          }
> >          case Event_ConnectionClosed::SERVER_SHUTDOWN: reasonStr = tr("Scheduled server shutdown."); break;
> > -        case Event_ConnectionClosed::USERNAMEINVALID: reasonStr = tr("Invalid username.\nYou may only use A-Z, a-z, 0-9, _, ., and - in your username."); break;
> > +        case Event_ConnectionClosed::USERNAMEINVALID: reasonStr = tr("Invalid username."); break;
>
> If unused, then trash it [image: :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cockatrice/Cockatrice/pull/1180/files#r33732908>.
>